content/docs/rfcs/51/README.md

```diff
@@ -35,16 +35,15 @@ This document also covers discovery of topic shards.
 # Named Sharding
 
 *Named sharding* offers apps to freely choose pubsub topic names.
-App protocols SHOULD follow the naming structure detailed in [23/WAKU2-TOPICS](/spec/23/).
+It is RECOMMENDED for App protocols to follow the naming structure detailed in [23/WAKU2-TOPICS](/spec/23/).
 With named sharding, managing discovery falls into the responsibility of apps.
 
-The default Waku pubsub topic `/waku/2/default-waku/proto` can be seen as a named shard available to all app protocols.
+The default Waku pubsub topic `/waku/2/default-waku` can be seen as a named shard available to all app protocols.
```

Comment:
The same comment as @jm-clius pointed out, for compatibility with the existing implementations, we should support both topics as the default topic:

- `/waku/2/default-waku/proto`
- `/waku/2/default-waku`

The same way Protocol labs do with the multiaddr protocol `ipfs` which is an alternate form for `p2p` for backward compatibility reasons.